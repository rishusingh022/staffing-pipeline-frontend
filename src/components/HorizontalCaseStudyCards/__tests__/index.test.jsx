// import React from 'react';
// import { render } from '@testing-library/react';
// import HorizontalCaseStudyCards from '..';

describe('Horizontal Case-Study Card', () => {
  it('should render correctly', () => {
    // const { asFragment } = render(<HorizontalCaseStudyCards />);
    // expect(asFragment()).toMatchSnapshot();
  });
});
