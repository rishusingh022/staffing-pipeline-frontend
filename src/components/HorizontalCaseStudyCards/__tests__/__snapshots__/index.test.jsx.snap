// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Horizontal Case-Study Card should render correctly 1`] = `
<DocumentFragment>
  <div
    class="case-study-card-container"
  >
    <div
      class="case-study-image"
    >
      <img
        alt="test-image"
        class="image-component"
        src="http://surl.li/fkwje"
      />
    </div>
    <div
      class="case-study-detail-box"
    >
      <div
        class="case-study-name text-base font-bold"
      />
      <div
        class="case-study-upload-date text-xs text-gray-500"
      >
        uploaded on 23/02/23
      </div>
    </div>
  </div>
</DocumentFragment>
`;
